### ISSUE TITLE
Help with scoring custom sequence

### ISSUE THREAD

**Issue by albustruong** [Illumina/SpliceAI#153]
Hi Kishore,

Thanks a lot for your effort building this tool.

I've been trying to score a custom sequence using your provided script:

from keras.models import load_model
from pkg_resources import resource_filename
from spliceai.utils import one_hot_encode
import numpy as np

input_sequence = 'CGATCTGACGTGGGTGTCATCGCATTATCGATATTGCAT'
# Replace this with your custom sequence

context = 10000
paths = ('models/spliceai{}.h5'.format(x) for x in range(1, 6))
models = [load_model(resource_filename('spliceai', x)) for x in paths]
x = one_hot_encode('N'*(context//2) + input_sequence + 'N'*(context//2))[None, :]
y = np.mean([models[m].predict(x) for m in range(5)], axis=0)

acceptor_prob = y[0, :, 1]
donor_prob = y[0, :, 2]

Unfortunately, I couldn't find any output in my working directory. I'm unsure whether it's because it didn't run successfully or because there were no scores to output so there's no output file. Can you help me with this?

Thanks a lot.

Albus.
